### ISSUE TITLE
Change iframe functionality on show page

### ISSUE THREAD

**Issue by rotated8** [emory-libraries/blacklight-catalog#490]
- [x] Remove buttons for physical/electronic versions of the iframe
- [x] Remove authentication logic from iframe url construction
- [x] Change the existing iframe to show physical item information, and to only show up if the item has physical item information
- [x] Add a new iframe below the previous one, to show electronic item information, and to only show up if the item has electronic item information

Note: Both iframes should show up of the item has physical and electronic information.

**Comment by bwatson78** [comment]
PR made: https://github.com/emory-libraries/blacklight-catalog/pull/506
![Screen Shot 2021-05-05 at 2.01.25 PM.png](https://images.zenhubusercontent.com/5e6f6c00c3241244877b4aea/b09c80a1-07c0-4346-85ec-4131c075e9f5)

**Comment by eporter23** [comment]
@lovinscari I am seeing the tables and iframes on test, but sometimes they take a second to load. Can you send me examples of ones that aren't loading? Also, please note that the work done to correct the Online vs At the Library facet is pending a full reindex of test, which may impact some of the tables you are expecting to see. Re: the Table of Contents link example that appears to be an incorrectly coded MARC record, we can discuss more in our meeting.

**Comment by bwatson78** [comment]
@eporter23 Can I reindex each of these?

**Comment by bwatson78** [comment]
@lovinscari and @eporter23 
I have reindexed both. 9937225867702486 went away because it has been deleted from Alma. 990006759420302486 is still showing no info in where to get it. I will bring into my local and analyze why. Shouldn't take long.

**Comment by bwatson78** [comment]
@lovinscari and @eporter23

For 990006759420302486, the availability API is pulling nothing for the library value (neither is the solr document). The response lacks all datafields with tags we look for (`AVA` and `AVE`). For the solr field, the only reason it's being treated as "At the Library" is because it meets the rule of not having either of the tags below and the leader's position 7 doesn't contain 'e', 'f', 'g', 'k', 'o', or 'r' and controlfield '008' position 24 doesn't contain 'o' or 's'.

In other words, it's working as expected.

**Comment by eporter23** [comment]
@lovinscari I am good to close this one because I think for these two examples, they were data problems. I'm not sure if you have others for us to investigate.
